near-plugins-derive/src/access_controllable.rs

```diff
@@ -90,6 +84,7 @@ pub fn access_controllable(attrs: TokenStream, item: TokenStream) -> TokenStream
             Permissions,
             Bearers,
             BearersSet { permission: #bitflags_type },
+            AclStorage,
```

Comment:
According to the rules, it is dangerous to the storage key order. Also, we already have deployed contracts on mainnet that use `acl` plugin, so it will complicate the migration.

To migrate the existing contracts we will just declare the contract structure like that :
```
#[derive(BorshDeserialize, BorshSerialize)]
pub struct CounterV0 {
    counter: u64,
    __acl: __Acl,
}
```

The `Permissions` and `Bearers` maps will be migrated automatically without any extra steps.